client/Slf4jGrablTracing.java

```diff
@@ -0,0 +1,110 @@
+package grabl.tracing.client;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.UUID;
+
+public class Slf4jGrablTracing implements GrablTracing {
+    private static final Logger LOG = LoggerFactory.getLogger(Slf4jGrablTracing.class);
+
+    private final GrablTracing innerTracing;
+
+    Slf4jGrablTracing(GrablTracing inner) {
+        innerTracing = inner;
+    }
```

Comment:
What's the use case of dumping tracing data to logs? Do we need this use case right now or in the foreseeable future?

Reply:
It's very useful for testing tracing locally, I've been using it extensively already.

Reply:
👍